auth0/src/main/java/com/auth0/android/authentication/storage/CredentialsManagerException.java

```diff
@@ -15,4 +15,14 @@
     CredentialsManagerException(String message) {
         super(message);
     }
+
+    /**
+     * Returns true when this Android device doesn't support the cryptographic algorithms used
+     * to handle encryption and decryption, false otherwise.
+     *
+     * @return whether this device is compatible with {@link SecureCredentialsManager} or not.
+     */
+    public boolean isDeviceIncompatible() {
```

Comment:
When users receive a `CredentialsManagerException` after calling "save" or "get" credentials methods, they can check whether the exception is recoverable or not by checking this `isDeviceIncompatible` flag.